docs/data/material/components/pickers/lab-date-and-time-pickers.md

```diff
@@ -0,0 +1,19 @@
+# Date & Time pickers
```

Comment:
Let's drop the ampersand here and give this our standard sentence case headline formatting.

```suggestion
# Date and time pickers
```